pages/search_cars_page.py

```diff
@@ -0,0 +1,52 @@
+import allure
+from pages.base_page import BasePage
+from utils.browser_helper import BrowserHelper
+from .locators.locators import SearchCarsPageLocators
+
+
+class SearchCarsPage(BrowserHelper, BasePage):
+    @allure.step("Accept cookies")
+    def accept_cookies(self):
+        try:
```

Comment:
it is better to check for visibility and click rather than putting in try block

Reply:
In the case when we use "expected wait" with visibility and if pop-up with cookies will not appear - we get TimeoutException. So, the test will be failed. Or maybe you mean to use if-else block?